recipes/debuglater/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "debuglater" %}
+{% set version = "1.4.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/debuglater-{{ version }}.tar.gz
+  sha256: 7633ce444f14fd1b27dffeb2374c59c2eee69cae7a4cfe951e7dfdc2b67cc4e6
+
+build:
+  entry_points:
+    - debuglater=debuglater.cli:main
+    - dltr=debuglater.cli:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - colorama
+    - python >=3.6
+
+test:
+  imports:
+    - debuglater
+  commands:
+    - pip check
+    - debuglater --help
+    - dltr --help
+  requires:
+    - pip
```

Can we run the testsuite?

```suggestion
  commands:
    - pip check
    - debuglater --help
    - dltr --help
    - pytest tests/
  requires:
    - pip
    - pytest
  source_files:
    - tests/
```

---

we run the tests on the main [repo](https://github.com/ploomber/debuglater/actions), I thought of skipping them here to keep runtime short

---

Hi @conda-forge/staged-recipes, any feedback here? Can we proceed with the existing recipe? We're running the test suite already in our CI system, so we can skip it here and save some build time.

---

Running full tests is not required for conda-forge packages. The primary purpose of the conda tests section is for testing that the packaging was successful. This does usually mean running some unit tests if the package is compiled, but for python packages imports, pip checks, and entry points are usually sufficient.